src/app/components/mno-password/mno-password.coffee

```diff
@@ -7,19 +7,21 @@ angular.module 'mnoEnterpriseAngular'
         data: '=',
         confirm: '=?',
         current: '=?',
+        minLength: '@',
         currentText: '@',
         newText: '@',
         confirmText: '@'
       },
       templateUrl: 'app/components/mno-password/mno-password.html',
       link: (scope, element, attrs) ->
         # Init args
+        scope.minLength = 6 if !scope.minLength
```

Comment:
Should the default min length values be aligned ? It is set to `6` here and `8` inside the `DEVISE_CONFIG`